.github/workflows/main.yml

```
@@ -13,7 +13,7 @@ jobs:
 
     strategy:
       matrix:
-        node-version: [14.x, 16.x]
+        node-version: [14.x, 16.x, 18.x]
```

```suggestion
        node-version: [18.x, 20.x]
```

Node 14 and 16 are both out of support now (Node 16 reached EOL on Sept 11) so there's no point in checking them in the matrix tests. Might as well start checking Node 20 while we're at it.